cognite_toolkit/_cdf_tk/prototypes/interactive_init.py

```diff
@@ -166,64 +174,67 @@ def interactive(
 
         print(f"  [{'yellow' if mode == 'overwrite' else 'green'}]Using directory [bold]{init_dir}[/]")
 
+        selected: dict[str, dict[str, Any]] = {}
+        if package:
+            if not available.get(package):
+                raise ToolkitRequiredValueError(
+                    f"Package {package} is not known. Available packages are {', '.join(available.keys())}"
+                )
+
+            selected[package] = available[package].get("modules", {}).keys()
+
         loop = True
         while loop:
```

Comment:
Any reason not to use break?
```suggestion
        while True:
```